app/Models/AchievementAuthor.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Models;
+
+use App\Support\Database\Eloquent\BaseModel;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\SoftDeletes;
+use Spatie\Activitylog\LogOptions;
+use Spatie\Activitylog\Traits\LogsActivity;
+
+class AchievementAuthor extends BaseModel
+{
+    use SoftDeletes;
+
+    use LogsActivity {
+        LogsActivity::activities as auditLog;
+    }
+
+    protected $table = 'achievement_authors';
+
+    protected $fillable = [
+        'achievement_id',
+        'user_id',
+        'task',
+        'created_at', // Used for backdating.
+        'updated_at',
+    ];
+
+    // == logging
+
+    public function getActivitylogOptions(): LogOptions
+    {
+        return LogOptions::defaults()
+            ->logOnly([
+                'achievement_id',
+                'user_id',
+                'task',
+            ])
+            ->logOnlyDirty()
+            ->dontSubmitEmptyLogs();
+    }
```

Comment:
There is no way to access this currently from any UI. However, it makes sense to go ahead and start recording the data.